CONTRIBUTING.md

```diff
@@ -3,11 +3,7 @@
 ### Getting and building
 
 ### Assumed
- * A working GCC (on mac os x something like `xcode-select --install` will get you started)
```

Comment:
We're not explicitly invoking the C compiler any more. I thought `brew install go` would handle installing a C compiler, but apparently it doesn't. I'll re-add this.